modules/instrument_builder/jsx/react.instrument_builder.js

```diff
@@ -17,6 +17,7 @@ var LoadPane = React.createClass({
       // This is used to alert the user if the file was
       // loaded successfully or there was an error with
       // the loading.
+      disabled: true,
```

Comment:
The comment above this is now wrong, the comment is for the alert, not the disabled attribute..

Reply:
Thanks. I moved it above the comments in my latest commit.